modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java

```diff
@@ -123,6 +123,12 @@ private List<InfoFieldValue<Object>> _getCommerceOrderInfoFieldValues(
 					_isOrderStatus(
 						commerceOrder,
 						CommerceOrderConstants.ORDER_STATUS_COMPLETED)));
+			commerceOrderInfoFieldValues.add(
+				new InfoFieldValue<>(
+					CommerceOrderInfoItemFields.completedOrderStatusInfoField,
+					_isOrderStatus(
+						commerceOrder,
+						CommerceOrderConstants.ORDER_STATUS_COMPLETED)));
```

Comment:
Hey @victorKammerer in the `InfoFieldValue` above we already have the map for `CommerceOrderConstants.ORDER_STATUS_COMPLETED`, I believe we can just replace `CommerceOrderInfoItemFields.companyIdInfoField` with `CommerceOrderInfoItemFields.completedOrderStatusInfoField`